src/gm3/components/map.js

```diff
@@ -498,7 +496,8 @@ class Map extends React.Component {
             // setup the spatial filter.
             query_params.geometryType = geom_type_lookup[queryGeometry.type];
             query_params.geometry = esri_format.writeGeometry(ol_geom);
-            query_params.spatialRel = 'esriSpatialRelIntersects'; // for lines?:'esriSpatialRelEnvelopeIntersects';
+            query_params.spatialRel = 'esriSpatialRelIntersects';
+            // for lines?:'esriSpatialRelEnvelopeIntersects';
```

Comment:
This should likely go above the query_params line.

Reply:
Really, it should be implemented or deleted.

Reply:
It's effectively a TODO: for an unimplemented case.

Reply:
It doesn't describe what the line above it does, it describes future work.  That's why I put it below.